src/components/Metrics/CustomMetrics.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { RouteComponentProps, withRouter } from 'react-router';
+import { Toolbar, ToolbarRightContent, FormGroup } from 'patternfly-react';
+
+import RefreshContainer from '../../containers/RefreshContainer';
+import * as API from '../../services/Api';
+import { KialiAppState } from '../../store/Store';
+import { DurationInSeconds } from '../../types/Common';
+import * as M from '../../types/Metrics';
```

Comment:
Personally, I prefer to not use short acronyms, but no change needed, just more a comment.